src/nemos/base_regressor.py

```diff
@@ -684,3 +685,56 @@ def _optimize_solver_params(self, X: DESIGN_INPUT_TYPE, y: jnp.ndarray) -> dict:
     def _get_optimal_solver_params_config(self):
         """Return the functions for computing default step and batch size for the solver."""
         pass
+
+    def save_params(self, filename: str = None) -> None:
+        """
+
+        Save parameters of the model to a npz file.
```

Comment:
```suggestion
        """
        Save parameters of the model to a npz file.
```
To match nemos conventions